sustainml_modules/sustainml_modules/sustainml-wp4/index.html

```diff
@@ -0,0 +1,285 @@
+        <select id="modality-select">
+            <option value="">Select...</option>
+            <option value="Image/video">Image/video</option>
+            <option value="Text">Text</option>
+            <option value="Sensor">Sensor</option>
+            <option value="Audio">Audio</option>
+        </select>
+        <div id="problem-type-section" class="hidden">
+            <p>What's your problem type?</p>
+            <select id="problem-type-select"></select>
+        </div>
+        <div id="evaluation-metrics-section" class="hidden">
+            <p>What are your evaluation metrics?</p>
+            <select id="evaluation-metrics-select"></select>
+        </div>
+        <div id="model-section" class="hidden">
+            <p>Model's suggestion?</p>
+            <select id="model-select"></select>
+        </div>
+        <div id="submit-pd" class="hidden">
+            <button class="response-button">Submit PD Info</button>
+        </div>
+        <div id="node-status" class="hidden">
+            <!-- JSON data will be displayed here -->
+        </div>
+        <div id="refresh-status" class="hidden">
+            <button class="status-button">Refresh Node Status</button>
+        </div>
+        <div id="framework-results" class="hidden">
+            <!-- JSON data will be displayed here -->
+        </div>
+        <div id="refresh-results" class="hidden">
+            <button class="results-button">Refresh Results</button>
+        </div>
+    </div>
+    <!-- Environment section -->
+    <div id="env-section" class="hidden">
+        <h2>Environment Information</h2>
+        <div class="environment-input">
+            <label for="hardware-input">What's your hardware?</label>
+            <input type="text" id="hardware-input" placeholder="e.g., CPU, GPU, RAM"/>
+        </div>
+        <div class="environment-input">
+            <label for="cloud-local-select">Where are you running the ML model?</label>
+            <select id="cloud-local-select">
+                <option value="">Select...</option>
+                <option value="Local">Local</option>
+                <option value="Cloud">Cloud</option>
+            </select>
+        </div>
+        <div class="environment-input">
+            <label for="location-input">What's your personal location?</label>
+            <input type="text" id="location-input" placeholder="e.g., City, Country"/>
+        </div>
+        <button id="submit-environment" class="response-button">Submit Environment Info</button>
+    </div>
+
+    <div id="navigation-overview" class="hidden">
+        <div class="circle-container small">
+            <div class="circle data-circle small-circle">Data</div>
+            <div class="circle pd-circle large-circle">PD</div>
+            <div class="circle env-circle small-circle">Environment</div>
+        </div>
+    </div>
+
+    <script>
+        document.getElementById('pd-section').style.display = 'none';
```

Comment:
Same comment would apply here

Reply:
Was not able to effectively migrate it to a separate script